pyinfraformat/tests/test_coordinates.py

```diff
@@ -11,6 +11,7 @@
     check_area,
     proj_espoo,
     proj_helsinki,
+    proj_porvoo,
```

Comment:
proj_finland_porvoo?

Reply:
nah. I think only one porvoo exists and 'porvoo' string in infrafromat file means this. 

![kuva](https://user-images.githubusercontent.com/44357059/79136570-e7d2e000-7db9-11ea-9f3f-209f390cba61.png)

Reply:
Sounds fine